conjur/logic/role_logic.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""
+RoleLogic module
+
+This module is the business logic for executing the ROLE command
+"""
+
+# Builtins
+import logging
+
+from conjur_api.errors.errors import HttpStatusError
+
+# pylint: disable=too-few-public-methods
+class RoleLogic:
+    """
+    RoleLogic
+
+    This class holds the business logic for executing and manipulating
+    returned data
+    """
+
+    def __init__(self, client):
+        self.client = client
+
+    # pylint: disable=logging-fstring-interpolation
+    def role_exists(self, kind: str, role_id: str) -> bool:
+        """
+        Method to handle all get role exists action activity
```

Comment:
Looks like this comment was copied from [variable_logic.py](https://github.com/cyberark/cyberark-conjur-cli/blob/a2f9f912aa9696b1bda81122745647c43fa9ac49/conjur/logic/variable_logic.py#L32) which was clearly not written by a native English speaker...

Maybe something like this would be clearer?
```suggestion
        Method to handle role exists action
```